java-squid/src/main/java/org/sonar/java/JavaAstScanner.java

```diff
@@ -24,13 +24,14 @@
 import org.sonar.java.ast.AstScanner;
 import org.sonar.java.ast.parser.JavaParser;
 import org.sonar.java.ast.visitors.CommentLinesVisitor;
-import org.sonar.java.ast.visitors.FileVisitor;
+import org.sonar.java.model.VisitorsBridge;
 import org.sonar.squidbridge.SquidAstVisitor;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.squidbridge.SquidAstVisitor'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)